src/jvm_wrapper/bootstrap.cpp

```diff
@@ -36,15 +45,11 @@ void Bootstrap::register_engine_type(JNIEnv* p_env, jobject p_this, jobjectArray
 
 Bootstrap::Bootstrap(jni::Env& p_env, jni::JObject p_wrapped) : JvmInstanceWrapper(p_env, p_wrapped) {}
 
-
 void Bootstrap::init(jni::Env& p_env, const String& p_project_path, const String& p_jar_file, const jni::JObject& p_class_loader) {
     LOCAL_FRAME(2);
     jni::JObject project_path = p_env.new_string(p_project_path.utf8().get_data());
     jni::JObject jar_file {p_env.new_string(p_jar_file.utf8().get_data())};
-    jvalue args[5] = {
-      jni::to_jni_arg(project_path),
-      jni::to_jni_arg(jar_file),
-      jni::to_jni_arg(p_class_loader)};
+    jvalue args[5] = {jni::to_jni_arg(project_path), jni::to_jni_arg(jar_file), jni::to_jni_arg(p_class_loader)};
```

Comment:
5? shouldn't it be 3?

(not from your PR but just noticed)

Reply:
True, I forgot to reduce the size of the array in the JVM loading PR after changing the signature of the method.